scripts/vfx/build_oiio.sh

```diff
@@ -15,6 +15,7 @@ cmake -DCMAKE_INSTALL_PREFIX=${ASWF_INSTALL_PREFIX} \
       -DOIIO_BUILD_TOOLS=OFF \
```

Comment:
Let's turn this on, too. Certainly the OSL CI will need this.

Reply:
Good to know, done!